templates/.gitignore

```diff
@@ -0,0 +1,7 @@
+## AsciiBinder-specific ignores
+_preview
+_package
+*.swp
```

Comment:
Nevermind...I didn't realize this was getting copied on create so it's for the *new* repo (it's so meta :D)